ui/pages/unlock-page/__snapshots__/unlock-page.test.js.snap

```diff
@@ -19,73 +19,63 @@ exports[`Unlock Page should match snapshot 1`] = `
           <div
-            class="MuiInputBase-root MuiInput-root MuiInput-underline TextField-materialUnderline-3 MuiInputBase-fullWidth MuiInput-fullWidth Mui-focused Mui-focused MuiInputBase-formControl MuiInput-formControl"
+            class="mm-box mm-text-field mm-text-field--size-lg mm-text-field--focused mm-text-field--truncate mm-form-text-field__text-field mm-box--padding-right-0 mm-box--padding-left-0 mm-box--display-inline-flex mm-box--align-items-center mm-box--background-color-background-default mm-box--rounded-lg mm-box--border-width-1 box--border-style-solid"
           >
             <input
-              aria-invalid="false"
-              autocomplete="current-password"
-              class="MuiInputBase-input MuiInput-input"
+              aria-label="Password"
+              autocomplete="on"
+              class="mm-box mm-text mm-input mm-input--disable-state-styles mm-text-field__input mm-text--body-md mm-box--margin-0 mm-box--padding-0 mm-box--padding-right-4 mm-box--padding-left-4 mm-box--color-text-default mm-box--background-color-transparent mm-box--border-style-none"
               data-testid="unlock-password"
-              dir="auto"
+              focused="true"
               id="password"
+              placeholder="Enter password to continue"
               type="password"
               value=""
             />
           </div>
         </div>
+        <button
+          class="mm-box mm-text mm-button-base mm-button-base--size-lg mm-button-base--disabled mm-button-base--block mm-button-primary mm-button-primary--disabled mm-text--body-md-medium mm-box--padding-0 mm-box--padding-right-4 mm-box--padding-left-4 mm-box--display-inline-flex mm-box--justify-content-center mm-box--align-items-center mm-box--color-primary-inverse mm-box--background-color-primary-default mm-box--rounded-pill"
+          data-testid="unlock-submit"
+          data-theme="light"
+          disabled=""
+          type="submit"
+        >
+          Unlock
+        </button>
       </form>
       <button
-        class="button btn--rounded btn-default"
-        data-testid="unlock-submit"
-        disabled=""
-        style="margin-top: 20px; height: 56px; font-weight: 500; box-shadow: none; border-radius: 100px;"
-        variant="contained"
+        class="mm-box mm-text mm-button-base mm-button-link mm-button-link--size-auto mm-text--body-md-medium mm-box--margin-bottom-6 mm-box--padding-0 mm-box--padding-right-0 mm-box--padding-left-0 mm-box--display-inline-flex mm-box--justify-content-center mm-box--align-items-center mm-box--color-primary-default mm-box--background-color-transparent"
+        data-testid="unlock-page-link"
       >
-        Unlock
+        Forgot password?
       </button>
-      <div
-        class="unlock-page__links"
-      >
-        <a
-          class="button btn-link unlock-page__link"
-          role="button"
-          tabindex="0"
-        >
-          Forgot password?
-        </a>
-      </div>
```

Comment:
We are changing the "Forgot password?" to a button element as it uses the onClick event handler and has no reason to use the link element